src/extensions/Statiq.Html/Globalization.cs

```diff
@@ -0,0 +1,531 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using AngleSharp.Dom;
+using AngleSharp.Dom.Html;
+using AngleSharp.Parser.Html;
+using Statiq.Common;
+
+namespace Statiq.Html
+{
+    public class Globalization : Module
```

Comment:
Is there anything about this module that would prevent from processing documents in parallel? If not, we could derive from ParallelModule instead and move the `HtmlParser` up to a class-level static.